tts/UI.def.lua

```diff
@@ -170,16 +197,38 @@ UI = {}
 ---@field children nil
 ---@field value nil | string
 
+---@shape tts__UITableLayoutElement : tts__UIElementBase
+
+---@shape tts__UIToggleGroupElement : tts__UIElementBase
+
+---@shape tts__UIToggleButtonElement_Attributes : tts__UIInputElementBase_Attributes
```

Comment:
My apologies if this was like this earlier and I missed it. However, Luanalysis requires a blank line between type definition blocks. Otherwise, it gets a bit confused.

In this particular case, seems as `tts__UIToggleButtonElement` doesn't appear to have any attributes of its own. You could just delete this line as well as line 206 (where the `attributes` field is overridden).

Reply:
No apologies required.  It wasn't that way, I don't think.  It think it was modified post-fixing InputElement things.  There are attributes that just haven't been added yet.    But  I will strike it.